.github/workflows/ci.yml

```
@@ -0,0 +1,17 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+  workflow_dispatch:
```

```suggestion
  pull_request:
```

(Sorry, forgot this one.) We probably want to have this ran on pull requests as well. (So that if someone makes a PR, this is ran in order to catch anything that may be wrong.)

Ref. [this](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#onpushpull_requestbranchestags)